INSTALL.md

```diff
@@ -68,49 +71,33 @@ Nix users can also get all the required dependencies by running:
 
     $ nix-shell
 
-Advanced users may want to experiment with the OCaml Flambda
-compiler as way to improve the performance of Coq. In order to
-profit from Flambda, a special build of the OCaml compiler that has
-the Flambda optimizer enabled must be installed. For OPAM users,
-this amounts to installing a compiler switch ending in `+flambda`,
-such as `4.12.1+flambda`. For other users, YMMV. Once `ocamlopt -config`
-reports that Flambda is available, some further optimization options
-can be used; see the entry about `flambda options` in the build guide
-for more details.
-
-Choice of Build and Installation Procedure
-------------------------------------------
-
-There are two partially overlapping infrastructures available to build
-Coq. They are available through `Makefile.make` (legacy / hybrid
-build) and `Makefile.dune` (full Dune build).
-
-You can use the `COQ_USE_DUNE` environment variable to change the one
-to use by default. This is useful for Coq development, where we
-recommend to rely mainly on `Makefile.dune`. Note that mixing the two
-systems is not perfectly supported and may lead to confusing behavior.
+Run-time dependencies of native compilation
+-------------------------------------------
 
-In both cases, the OCaml parts are built using
-[Dune](https://github.com/ocaml/dune). The main difference between the
-two systems is how the `.vo` files are built.
+The OCaml compiler and findlib are build-time dependencies, but also
+run-time dependencies if you wish to use the native compiler.
 
-In the case of `Makefile.make`, `.vo` files are built with a legacy
-Makefile, similar to what `coq_makefile` would do. In the case of
-`Makefile.dune`, `.vo` files are built with Dune, thanks to its
-recently-added, and still experimental, Coq mode.
+Build and install procedure
+---------------------------
 
-See the documentation for the two infrastructures:
+Note that Coq supports a faster, but less optimized developer build,
+but final users must always use the release build. See
+[dev/doc/build-system.dune.md](dev/doc/build-system.dune.md)
+for more details.
 
-- [Legacy build](dev/doc/INSTALL.make.md)
-- [Full Dune build](dev/doc/build-system.dune.md)
+To build and install Coq (and CoqIDE if desired) do:
 
-See also [`dev/doc/README.md`](dev/doc/README.md).
+    $ ./configure -prefix <install_prefix> $options
+        $ make dunestrap
+        $ dune build -p coq-core,coq-stdlib,coq,coqide-server,coqide
+    $ dune install --prefix=<install_prefix> coq-core coq-stdlib coq coqide-server coqide
 
-Run-time dependencies of native compilation
--------------------------------------------
+You can drop the coqide packages if not needed.
```

Comment:
nit: CoqIDE rather than coqide

Reply:
used a quote instead as the packages are lowercase